packages/aws-cdk-lib/aws-s3/README.md

```diff
@@ -624,10 +624,17 @@ enable the`autoDeleteObjects` option.
 
 When `autoDeleteObjects` is enabled, `s3:PutBucketPolicy` is added to the bucket policy. This is done to allow the custom resource this feature is built on to add a deny policy for `s3:PutObject` to the bucket policy when a delete stack event occurs. Adding this deny policy prevents new objects from being written to the bucket. Doing this prevents race conditions with external bucket writers during the deletion process.
 
+Pass a custom log group via the `autoDeleteObjectsLogGroup` option, which will be used by the custom resource lambda.
+
 ```ts
+import { ILogGroup } from 'aws-cdk-lib/aws-logs';
+
+declare const logGroup: ILogGroup;
```

Comment:
nit: I think its more valuable if you actually define a log group here. The log group from the integ test should do.

```suggestion
const logGroup = new logs.LogGroup(this, 'LogGroup', {
        logGroupName: 'AutoDeleteObjectsLambdaLogs',
        retention: logs.RetentionDays.THREE_DAYS,
      }),
```

Reply:
My first revision was defining the log group explicitly. But I found the [Lambda REAME](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda-readme.html#advanced-logging) that uses this style. But I agree defining the log group is more valuable to the reader. Will update